contracts/contracts/FleekERC721.sol

```diff
@@ -15,18 +15,13 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
     using FleekStrings for FleekERC721.App;
     using FleekStrings for FleekERC721.AccessPoint;
     using FleekStrings for string;
+    using FleekStrings for uint24;
 
-    event NewBuild(uint256 indexed tokenId, string commitHash, address indexed triggeredBy);
-    event NewTokenName(uint256 indexed tokenId, string name, address indexed triggeredBy);
-    event NewTokenDescription(uint256 indexed tokenId, string description, address indexed triggeredBy);
-    event NewTokenLogo(uint256 indexed tokenId, string logo, address indexed triggeredBy);
-    event NewTokenExternalURL(uint256 indexed tokenId, string externalURL, address indexed triggeredBy);
-    event NewTokenENS(uint256 indexed tokenId, string ENS, address indexed triggeredBy);
-    event NewTokenColor(uint256 indexed tokenId, uint24 color, address indexed triggeredBy);
-
+    event MetadataUpdate(uint256 indexed _tokenId, string key, string value, address indexed triggeredBy);
```

Comment:
Should we have multiple overided functions where the value param has different types. Wondering if there is an efficiency gain if we move to that when solidity is packaging up the data to be sent to the event.
Example 
```
 event MetadataUpdate(uint256 indexed _tokenId, string key, uint24 value, address indexed triggeredBy);
 event MetadataUpdate(uint256 indexed _tokenId, string key, string value, address indexed triggeredBy);
 ...
```